modules/serverless-nodejs-functions-context-objects.adoc

```diff
@@ -8,12 +8,6 @@
 
 Functions are invoked by providing a `context` object as the first parameter. This object provides access to the incoming HTTP request information.
 
-.Example context object
```

Comment:
Hi @ochromy . The similiar example is intended in the CloudEvent data section?

Reply:
Hey @dfitzmau . The second example makes more sense in its context. It would be also possible to share the _context_ object and _data_ parameter descriptions between the two modules but for now I only removed the first example as it was deemed redundant with the signature below.